main.cpp

```diff
@@ -125,84 +175,110 @@ struct CarWash
-    3) Deliver products
-*/
-
 struct Market
 {
-    //number of products in local inventory
-    float numProdLocalInv = 215.3f;
-    //number of vehicles used in logistics
+    double numProdLocalInv = 215;
     int numLogisticVehicles = 5; 
-    //number of people working in store
     int numPeopleWorkingAtStore =32; 
-    //amount of utilities
-    float basicUtilities = 320.53f;  
-    //daily income
-    float dailyIncome = 2750; 
+    double dailyBasicUtilitiesFee = 320.53;  
+    double dailyIncome = 0; 
+    double dailyProfit = 0;
 
     struct Customer
     {
         int age = 38;
         bool member = true;
         int visitsPerWeek = 7;
         std::string gender = "Male";
-        std::string transportation = "Scooter";
-
-        void eatFood(int Frequency = 5);
-        float calculateWorkedTime(bool atHome = true, 
-                                  float time = 8.5f); 
-        float computeMonthlyExpenses(float rent = 1500,  
-                                     float food = 450, 
-                                     float fun = 120,
-                                     float other = 230);
+        int breakfastProds = 4;
+        int morningBreakProds = 2;
+        int lunchProds = 5;
+        int coffeeBreakProds = 2;
+        int dinnerProds = 3;
+        int productsToOrder;
+        double productsPrice;
+        double totalToPay;
+
+        void selectDailyFood(); 
+        void calculateOrderPrice(); 
+        void orderProducts(bool deliveryRequired = true);
     };
 
-    //Sell daily use products
-    void sellProducts(Customer Nick); 
-    //Help customers with any issue
-    void solveCustomerNeeds(Customer Jack, bool solved = true, bool quest = false); 
-    //Deliver products
-    void deliverProducts(Customer Phil, bool success = true, int time = 35);
+    Customer nick;
+    Customer jack;
+
+    void sellProducts();
+    void adjustInventary();
+    void computeDailyProfit();
 };
 
-/*
-Thing 2) University
-5 properties:
-    1) number of classrooms (int)
-    2) number of laboratories (int)
-    3) number of professors (int)
-    4) amount of semestral income (float)
-    5) classes taken per period (int)
-3 things it can do:
-    1) Teach students
-    2) Do collaborate research 
-    3) Perform cultural activities
- */
+void Market::Customer::selectDailyFood()
+{
+    productsToOrder = breakfastProds + morningBreakProds + lunchProds +coffeeBreakProds + dinnerProds;
+}
+
+void Market::Customer::calculateOrderPrice()
+{
+    double bfProdsPrice = 0.75;
+    double mbProdsPrice = 1.25;
+    double lProdsPrice = 0.85;
+    double cbProdsPrice = 1.25;
+    double dProdsPrice = 0.9;
+    double memberDiscount = 0.9;
+    productsPrice = (breakfastProds * bfProdsPrice) +\
+                    (morningBreakProds * mbProdsPrice) +\
+                    (lunchProds * lProdsPrice) +\
+                    (coffeeBreakProds * cbProdsPrice) +\
+                    (dinnerProds * dProdsPrice);
```

Comment:
remove the `\`.  It is not necessary and just adds visual confusion to the code.
Semicolons indicate the end of a statement in C++, not newlines.
- [ ] revise